includes/entities/class-fs-site.php

```diff
@@ -10,15 +10,15 @@
         exit;
     }
 
-    /**
-     * @property int $blog_id
-     */
-    #[AllowDynamicProperties]
     class FS_Site extends FS_Scope_Entity {
         /**
          * @var number
          */
         public $site_id;
+        /**
+         * @var number
```

Comment:
@DanieleAlessandra This should be `int` like in the `get_unfiltered_site_url()` method. For entity IDs though, we use `number` (like in the `FS_Entity` class).